templates/travel/partials/search-navbar/search-navbar.snip.html

```diff
@@ -0,0 +1,17 @@
+<section class="relative z2">
```

Comment:
copyright

Reply:
Type `Done` in the comments that you have addressed so that i know that you have addressed those (GH makes this much harder for us - Sorry :))